examples/lib/intermediateHtmlStepUi.js

```diff
@@ -66,20 +66,20 @@ function IntermediateHtmlStepUi(_sequencer, step, options) {
 
 
   function selectNewStepUi() {
-    var insertSelect = $(step.ui.querySelector('.insert-step-select'));
+    var insertSelect = $step('.insert-step-select');
     var m = insertSelect.val();
-    $(step.ui.querySelector('.insertDiv .info')).html(_sequencer.modulesInfo(m).description);
-    $(step.ui.querySelector('.insertDiv .add-step-btn')).prop('disabled', false);
+    $step('.insertDiv .info').html(_sequencer.modulesInfo(m).description);
+    $step('.insertDiv .add-step-btn').prop("disabled", false);
   }
     
     
   var toggleDiv = function(callback = function(){}){
-    $(step.ui.querySelector('.insertDiv')).collapse('toggle');
-    if ($(step.ui.querySelector('.insert-text')).css('display') != 'none'){
-      $(step.ui.querySelector('.insert-text')).fadeToggle(200, function(){$(step.ui.querySelector('.no-insert-text')).fadeToggle(200, callback);});
+    $step('.insertDiv').collapse('toggle');
+    if ($step('.insert-text').css('display') != "none"){
```

Comment:
why double-quotes here(" ") ? Any specific reason? if not can we keep them single?

Reply:
No specific reason, It was there before(I guess) but eslint did nothing

Reply:
Nope we had single quotes that is why I was wondering why do we have them in this file!

Reply:
Nope we had single quotes that is why I was wondering why do we have them in this file!

Reply:
Should I change this?

Reply:
@aashna27 shall I push another commit fixing this? This must be some old code. By any chance, if you are able to push a commit, it will be helpful to me if you do so as I am currently busy solving math 📚+-÷×π=%....

Reply:
Ty

Reply:
```suggestion
    if ($step('.insert-text').css('display') != 'none'){
```